ARCHITECTURE.md

```diff
@@ -0,0 +1,116 @@
+# Architecture of Git for Windows
+
+Git for Windows is a complex project.
+
+First and foremost, it is a friendly fork of [https://github.com/git/git](`git/git`), aiming to improve Git's Windows support. The [https://github.com/git-for-windows/git](`git-for-windows/git`) repository contains dozens of topics on top of `git/git`, some awaiting to be "upstreamed" (i.e. to be contributed to `git/git`), some still being stabilized, and a few topics are specific to the Git for Windows project and are not intended to be integrated into `git/git` at all.
```

Comment:
Markdown link syntax is the other way around. 

```markdown
[Description](https://example.com)
```

Reply:
D'oh. Thank you!